src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -11,18 +11,24 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
+    public String addIndentation(String str) {
+        str = str.replaceAll("Parameters:", "    Parameters:");
+        str = str.replaceAll("Example:", "        Example:");
+        return str;
+    }
+
     @Override
     public CommandResult execute() {
         return new CommandResult(
-                AddCommand.MESSAGE_USAGE
-                + "\n" + DeleteCommand.MESSAGE_USAGE
-                + "\n" + ClearCommand.MESSAGE_USAGE
-                + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
-                + "\n" + ViewCommand.MESSAGE_USAGE
-                + "\n" + ViewAllCommand.MESSAGE_USAGE
-                + "\n" + HelpCommand.MESSAGE_USAGE
-                + "\n" + ExitCommand.MESSAGE_USAGE
+                addIndentation(AddCommand.MESSAGE_USAGE)
```

Comment:
Maybe a more flexible way will be to have `addIndentation` take in an indentation level, and then it moves the string to that indentation level.